nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/RunnerFeature.java

```diff
@@ -101,12 +101,26 @@ default void stop(FeaturesRunner runner) throws Exception {
     default void beforeSetup(FeaturesRunner runner) throws Exception {
     }
 
+    /**
+     * Before entering in the @Before methods
+     */
```

Comment:
Since it's public shouldn't it (and `#beforeSetup`) be annotated since 11.1 ?

Reply:
Indeed, but this is a backport of methods already present on master.